examples/stable_diffusion/stable_diffusion_docker.yaml

```diff
@@ -1,23 +1,24 @@
 #SkyPilot YAML to run stable diffusion web tool on 1 V100 GPU.
 
 resources:
+  cloud: gcp
```

Comment:
No need for gcp as we are using DockerHub?

Reply:
Multi-gpu was not supported by the underlying repos we used to run stable diffusion, so I don't think there is a need to support multi-GPU configuriation as of now.